test/svm/SvmSpoke.Bundle.ts

```diff
@@ -1732,4 +1732,161 @@ describe("svm_spoke.bundle", () => {
       assert.include(err.toString(), "Invalid Merkle proof", "Expected merkle verification to fail");
     }
   });
+
+  describe("Execute Max multiple refunds with claims", async () => {
+    const executeMaxRefundClaims = async (testConfig: { solanaDistributions: number; useAddressLookup: boolean }) => {
+      // Add leaves for other EVM chains to have non-empty proofs array to ensure we don't run out of memory when processing.
+      const evmDistributions = 100; // This would fit in 7 proof array elements.
+
+      const refundAddresses: web3.PublicKey[] = []; // These are relayer authority addresses used in leaf building.
+      const claimAccounts: web3.PublicKey[] = [];
+      const tokenAccounts: web3.PublicKey[] = [];
+      const refundAmounts: BN[] = [];
+      const initializeInstructions: TransactionInstruction[] = [];
+      const claimInstructions: TransactionInstruction[] = [];
+
+      for (let i = 0; i < testConfig.solanaDistributions; i++) {
+        // Create the token account.
+        const tokenOwner = Keypair.generate().publicKey;
+        const tokenAccount = (await getOrCreateAssociatedTokenAccount(connection, payer, mint, tokenOwner)).address;
+        refundAddresses.push(tokenOwner);
+        tokenAccounts.push(tokenAccount);
+
+        const [claimAccount] = PublicKey.findProgramAddressSync(
+          [Buffer.from("claim_account"), mint.toBuffer(), tokenOwner.toBuffer()],
+          program.programId
+        );
+
+        // Create instruction to initialize claim account.
+        initializeInstructions.push(await program.methods.initializeClaimAccount(mint, tokenOwner).instruction());
+        claimAccounts.push(claimAccount);
+
+        refundAmounts.push(new BN(randomBigInt(2).toString()));
+
+        // Create instruction to claim refund to the token account.
+        const claimRelayerRefundAccounts = {
+          signer: owner,
+          initializer: owner,
+          state,
+          vault,
+          mint,
+          tokenAccount,
+          claimAccount,
+          tokenProgram: TOKEN_PROGRAM_ID,
+          program: program.programId,
+        };
+        // Will not claim for the first relayer as if its token account was blacklisted.
```

Comment:
Out of curiosity, why do we simulate the first relayer being blacklisted?

Reply:
agree, this was rather arbitrary. now simplified the tests by claiming to all accounts